### ISSUE TITLE
Request: Choice of aosp-clang version

### ISSUE THREAD

**Issue by Lyahi** [grm34/ZenMaxBuilder#739]
Hi dev, A practical option would be to add an option to select the aosp-clang version.

Best.

**Comment by grm34** [comment]
Hey, you can use any tc/version you like, see `etc/user.cfg`

**Comment by Lyahi** [comment]
Do you mean assigning the file address 
of a specific clang in a URL variable _AOSP_CLANG_URL_?
If so, for a specific version I can assign and it will be fine. I meant listing the available versions from the branch in the console and assigning the selection to a variable. 🙂

**Comment by grm34** [comment]
Yes, change the url to clone the whole repo and switch branch/tag when you want for example. I keep this open, aosp_clang/llvm prebuilts for Q/R/S/T could be added if time.

**Comment by Lyahi** [comment]
Ok, I get it. ;)
